src/styles/bar.css

```diff
@@ -10,6 +10,17 @@
   font-size: 1rem;
 }
 
+@media (max-width: 64em) {
+  [role=banner] {
```

Comment:
So, no need to have the double selector anymore? Good thing if so 👍

Reply:
I don't understand why we need double selector before, it's ok in banks with this.

Reply:
Do you try on a native mobile browser? @GoOz seems to say that it was specifically for that case

Reply:
It was clearly a conflict between Cozy-bar styles and UI styles on mobile browsers when minimal-UI was deployed. 
I don't remember clearly what was the bug but it sure be cool to test this on native browser on mobile indeed.